src/tests/ie_test_utils/functional_test_utils/layer_tests_summary/run_parallel.py

```diff
@@ -0,0 +1,264 @@
+# Copyright (C) 2018-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+from utils import utils
+from argparse import ArgumentParser
+from subprocess import Popen, STDOUT, TimeoutExpired
+
+import os
+import sys
+import threading
+from hashlib import sha256
+from pathlib import Path
+from shutil import rmtree
+
+import csv
+
+import datetime
+
+if sys.version_info.major >= 3:
+    import _thread as thread
+else:
+    import thread
+
+FILENAME_LENGTH = 255
+
+TEST_TIMEOUT = 60000
```

Comment:
Maybe it will be better to add such hardcoded variables like this as a command line argument with default value.
In future it can allow to change this value for some specific test cases.

Reply:
of course it will be command-line :)

Reply:
![image](https://user-images.githubusercontent.com/22008923/207249224-92098777-02f5-4e54-ad51-d64bbab6d3ac.png)